src/main/host/tracker.c

```diff
@@ -429,12 +429,13 @@ static void _tracker_logNode(Tracker* tracker, LogLevel level, SimulationTime in
     if(!tracker->didLogNodeHeader) {
         tracker->didLogNodeHeader = TRUE;
         logger_log(logger_getDefault(), level, __FILE__, __FUNCTION__, __LINE__,
-                "[shadow-heartbeat] [node-header] "
-                "interval-seconds,recv-bytes,send-bytes,cpu-percent,delayed-count,avgdelay-milliseconds;"
-                "inbound-localhost-counters;outbound-localhost-counters;"
-                "inbound-remote-counters;outbound-remote-counters "
-                "where counters are: %s", _tracker_getCounterHeaderString()
-        );
+                   "[shadow-heartbeat] [node-header] "
+                   "interval-seconds,recv-bytes,send-bytes,cpu-percent,delayed-"
```

Comment:
Could we add a line break after a comma rather than breaking up the `delayed-count` key?

Reply:
Hmm, clang-format being a little too clever. I'll see if there's a global setting where I can disable this. Otherwise I'll just turn it off locally for this line.

Reply:
Done. (Exempted these lines from clang-format)